CleffyNotes/ViewControllers/KeysAndScalesViewController.swift

```diff
@@ -34,6 +37,8 @@ class KeysAndScalesViewController: UIViewController {
     
     @IBOutlet weak var flatSharpSegmentedControl: UISegmentedControl!
     
+    @IBOutlet weak var playButton: UIButton!
+    
     var scale: [Note]? = [Note(sharpName: "A", flatName: "A", frequency: 220.00),
                          Note(sharpName: "B", flatName: "B", frequency: 246.94),
                          Note(sharpName: "C\u{266F}", flatName: "D\u{266D}", frequency: 277.18),
```

Comment:
Hey, I see some unicode characters! Also some very specific frequencies that you had to look up.